src/main.ts

```diff
@@ -14,6 +14,8 @@ const main = async () => {
   const server = await createServer(mongoose, aggregates)
   const app = Express()
   server.applyMiddleware({ app })
+
+  app.enable('trust proxy')
```

Comment:
what does this do?

Reply:
This enables the `trust proxy` express setting. This is taken into effect when reading the `req.ip` field. Without that, the IP will always be taken straight from the request which would not work very well since Orion is behind a reverse proxy and that IP would always be the one of the proxy. With this enabled, `req.ip` will be populated with the value from the `x-forwarded-for` header instead

Reply:
Clear now, thanks!